identity-service/src/solanaNodeRegistration.js

```diff
@@ -0,0 +1,71 @@
+const Bull = require('bull')
+const config = require('./config.js')
+
+const JOB_FREQUENCY_MS = 15 /* min */ * 60 /* sec */ * 1000 /* msec */
+
+const registerNodes = async (audiusLibs, logger) => {
+  logger.info('Beginning node registration job')
+  const nodes = await audiusLibs.ServiceProvider.discoveryProvider.serviceSelector.getServices({ verbose: true })
+  const unregistered = []
+  for (const node of nodes) {
```

Comment:
just for curiosity, were you envisioning that this node registration thing be able to be used for other node registration? i think URSM will have something very similar. cc @jowlee

Reply:
I was not - we definitely could adapt it though